modules/dirwatcher.js

```diff
@@ -0,0 +1,44 @@
+import EventEmitter from 'eventemitter3';
+import fs from 'fs';
+import { differenceWith, isEqual } from 'lodash';
+
+export default class DirWatcher extends EventEmitter {
+  constructor() {
+    super();
+    this.filesData = [];
+  }
+
+  watch(dirPath, delay) {
+    const watchFiles = () => {
```

Comment:
Export this function, please

FYI: It would be better for further testing. For now we couldn't get access for it.